src/index.html

```diff
@@ -98,6 +98,9 @@ <h1>TECHNOLOGIES</h1>
     <h1>THANK YOU</h1>
     <p>YOU be the shit.</p>
     <h2>To Zoe, Greg, and Michael</h2>
+    <p>Thank you for helping us establish a solid foundation in the world of Software Development. Through all the Git conflicts, pits of despair, endless questions, screen share sessions, pinterest projects, pies, laughs, and everything in between, we are forever grateful for your time and concerted efforts to pass on your knowledge. You were instrumental in our growth over the past year, and without the fundamentals and patience you taught us, we would not be where we stand today as developers.  </p>
+    <h2>To Nathan, Martin, and Jameka</h2>
+    <p>Thanks persons two</p>
     <p>Thanks persons one</p>
```

Comment:
this placeholder line needs to be removed please!

Reply:
done. Thanks!